Jenkinsfile

```
@@ -236,8 +237,11 @@ def buildWindows(nodeVersion, arch) {
   return {
     myNode('windows && nodejs') {
       unstash 'source'
-
+      bat "nodist add ${nodeVersion}"
+      bat "nodist npm add ${npmVersion}"
```

```suggestion
      bat """
        nodist add ${nodeVersion}
        nodist npm add ${npmVersion}
      """
```
You can use a single shell block to run both commands.

---

Thanks for the suggestion. I like the simplicity of separating these. In my opinion it makes it simpler to determine what command failed.